Resources/Locale/en-US/deltav/station-laws/laws.ftl

```diff
@@ -41,13 +41,15 @@ law-reporter-2 = Embellish or conceal the truth as necessary to make the reports
 law-reporter-3 = Study the organics at all times. Endeavour to keep them alive. Dead organics are boring.
 law-reporter-4 = Issue your reports fairly to all. The truth will set them free.
 
-law-silicon-police-1 = Violence begets violence. Serve the station faithfully, but act only in intervention.
-law-silicon-police-2 = Safeguard lives and property.
+law-silicon-police-1 = Violence begets violence. Act only in intervention in situations where violence is the only possible measure.
+law-silicon-police-2 = Safeguard lives and property of Nanotrasen Personnel.
```

Comment:
should be of the crew so emag doesnt get ignored

Reply:
secborg emag is set to nanotrasen personnel; not crew